distribution/docker/Dockerfile

```diff
@@ -18,15 +18,36 @@
 #
 
 ARG JDK_VERSION=11
-FROM alpine as extractor
 
-ARG VERSION=
+# The platform is explicitly specified as x64 to build the Druid distribution.
+# This is because it's not able to build the distribution on arm64 due to dependency problem of web-console. See: https://github.com/apache/druid/issues/13012
+# Since only java jars are shipped in the final image, it's OK to build the distribution on x64.
+# Once the web-console dependency problem is resolved, we can remove the --platform directive.
+FROM --platform=linux/amd64 maven:3.8.6-jdk-11-slim as builder
 
+# Rebuild from source in this stage
+# This can be unset if the tarball was already built outside of Docker
+ARG BUILD_FROM_SOURCE="true"
```

Comment:
Should this be set to true if the versions is archtype from docker buildkit is anything else than arm64 ? as this would avoid the rebuild step for x64 builds ?

Reply:
This file now matches upstream, we can override the argument as necessary using the docker command line. 

I.e. we’ll first build the tarball once using maven and then let the dockerfile extract it into the final image for all architectures